runtime/src/main/java/com/dylibso/chicory/runtime/InterpreterMachine.java

```diff
@@ -1696,14 +1682,12 @@ private static void I64_LOAD32_U(MStack stack, Instance instance, Operands opera
     private static void I64_LOAD32_S(MStack stack, Instance instance, Operands operands) {
         var ptr = readMemPtr(stack, operands);
         var val = instance.memory().readI32(ptr);
-        // TODO this is a bit hacky
```

Comment:
curious: what was this about? 😬

Reply:
I don't remember :shrug: an early day comment that got copy pasted around ...